### ISSUE TITLE
Dirichlet theorem and the PNT

### ISSUE THREAD

**Issue by Rami5743** [leios/SoME_Topics#105]
## About the author

My name is Rami Aizenbud. I am a mathematician working in the Weizmann Institute of Since. You can find more information on my website: http://aizenbud.org/

## Quick Summary
This is about 2 related topics
The Dirichlet theorem - https://en.wikipedia.org/wiki/Dirichlet%27s_theorem_on_arithmetic_progressions 
and the PNT - https://en.wikipedia.org/wiki/Prime_number_theorem.

My first choice is probably Dirichlet theorem, but I am flexible. 
## Target medium
video/videos in the stile of 3b1b
### More details
I have written a Wikipedia article about the Dirichlet  Theorem in the Hebrew Wikipedia: See https://he.wikipedia.org/wiki/%D7%9E%D7%A9%D7%A4%D7%98_%D7%93%D7%99%D7%A8%D7%99%D7%9B%D7%9C%D7%94
I think this is a very exiting topic which is very suitable for animation. 
The prove involves lots of ideas from all over math, and many f these ideas can be illustrated.
On the other hand it is rather deep topic so it is quite challenging to make it accessible. Nevertheless, I explain the proof to my 10 years old kid, and I thin he understand a big part of it.

### Contact details

you can e-mail me on aizenr@gmail.com, and we can set a zoom meeting.

**Comment by 3b1b** [comment]
Hi Rami,

Dirichlet's theorem is indeed a very interesting topic, but getting it to be accessible might be a fun challenge. Can you add any details on exactly what approach you had in mind? Maybe even an outline or storyboard describing some of the core mini lessons that would have to be involved along the way?

**Comment by Rami5743** [comment]
Here is a possible plan:

Video 1. general introduction to the topic::
1. formulation + motivation
3. recollection of Euclid's proof 
4. optional: discuss its (in)effectivity.
5. optional: Discus Euclid like proof for special cases of Dirichlet theorem.
6. From Euclid's proof to (simplified) Erdos' proof (i.e. to count number of all numbers from 1 to n assuming there are only k primes in this interval)
7. optional: discuss its (relative) effectivity.
8. Euler product and diversion of the sum 1/p.

This lecture can serve also as a joint introduction to PNT and Dirichlet theorem. In this case item 4 is less relevant, but items 3, 6 become more relevant. Also, in this case, one can add Mertens'  (very simple) proof that \sum ln(p)/p \sim \sum 1/p which serves as a heuristic for PNT. If we add all the optional points we might have to split the video into 2.

Video 2.   Diriclet argument for simple special case(s):
Let d=3. Let A= \sum1/p for p=1 mod 3 and B= \sum1/p for p=2 mod 3. We know A+B=\infty. We need to prove |A-B|<\infty. This can be deduce from 0<L(1,\chi)<\infty, where \chi is the non-trivial character of \Z/3^*.  The later is very easy computation.
Cases d=4,6 is similar, d=5 is not much harder. d=7 is more problematic (see item 3 in the next video).

Video 3.   Reduction to L(1,\chi)=0:
1. Fourier transform on abelian groups -  there several possible arguments, one have to choose. The proof of Fourier transform is a computation for a given modulus, so maybe we can postpone it to another video (not necessarily part of this sires).
2. The convergence of L(1,\chi). Here also there several possible arguments, I have a favorite one, for which I made an animation. 
3. There is one more delicate point that often is swept under the rug, one has to decide whether to do it here (I prefer not to).
4. emphasize that now we have an argument that reduce the theorem for a particular progression to a computation, 
5. optional - we can give the formula $$L(1,\chi)=-\frac{1}{m}\sum_{k=1}^{m-1} \ln(1-e^{2\pi i k/m})\sum_{l=1}^{m}\chi(l)e^{2\pi i kl/m}$$ 
It is not useful for the general proof but useful for special cases

We can stop at this point, declaring that we won, since we made a very hard problem, into a computation that one can easily perform with  a computer till, say, d=1000. However we can also finish the proof

Video 4.   completion of the solution:
1. Dirichlet (simple) argument for the non-real case. I made a nice animation here to explain the asymptotic of \zeta near 1.
2. Optional - we can  squeeze this lemon to its end. Namely, we can deduce Dirichlet theorem for many progressions (base is non-square modulo any factor of the difference). We can also deduce the Dirichlet theorem for all progressions except those whose difference is a multiplier of one number. If we add this item we might have to postpone the next ones to another video.
3. optional - non-elementary, but very short Landau's proof.
4. The elementary Merten's analytic proof. Here, one have to study the asymptotes of the defining sires of  \zeta(1/2). I made another animation for it.
5. Optional - discussion of the more algebraic proofs6. Optional - Explanation that in all of the above approaches, one can avoid using item 1.

I'll be happy to discus any of it on zoom.

**Comment by ajayberriman** [comment]
Hi Rami,

I'm an English maths student about to go to university. I love writing and talking about maths and I feel lucky to have had teachers and fellow students who use their empathy to make maths accessible and most importantly to make it feel good to participate in. I'd like to think I can make people feel that way too.

I know it's possible because I find it hard to parse the answer to [Meaning of Rays in Polar Plot of Prime Numbers](https://math.stackexchange.com/questions/885879/meaning-of-rays-in-polar-plot-of-prime-numbers) but [Why do prime numbers make these spirals?](https://youtu.be/EK32jo7i5LQ) presents the same answer in an accessible way that I find feels good to think about.

Hearing that your 10-year-old was able to access Dirichlet is very encouraging because I heard how tricky it would be to talk about in @3b1b's video. Your plan to work up from Euclid is particularly interesting because it is a proof that I believe all A-Level maths students learn (the qualification before university in England). With this knowledge, perhaps another plan could be to use Dirichlet as a way to talk about some mathematical proofs having more power than others?

I think this would be interesting and useful to students like myself who have studied maths for most of our lives but still wonder what career mathematicians do. I find that my mathematical education can feel like it's been a pipeline where children become skilled adults who answer mathematical questions but don't ask them - for example, some of us make the important decision to invest our only student loan into a maths degree without asking ourselves (and then others) what our career will actually be like. Perhaps we could ask 'Why mathematicians keep asking questions about primes' because we would be talking about proofs and why it is significant that they keep getting more powerful.

Assuming that we could make Erdős and Dirichlet accessible, let me propose a plan - that's a very big assumption for me so I'm aware this could be overly simplistic (or worse :sweat_smile:).


# Why mathematicians keep asking questions about primes


## Euclid's proof

![Euclid's theorem](https://user-images.githubusercontent.com/17185640/173198693-03ea2b89-aaee-46d7-9f7b-9b9362e75a28.png)

If we use Euclid's theorem as our first point, it should be either familiar or simple enough to encourage the audience to start thinking about primes. We could explain that Euclid proved that any finite set of primes can be used to find at least another prime - we can generalise this idea later. In A-Level maths, Euclid's proof is taught as a proof by contradiction which I have been surprised to learn is not what Euclid actually wrote in _Elements_ - we can talk about the power of direct and indirect proof later, too.


## Generalising Euclid's proof

![Generalising Euclid's theorem](https://user-images.githubusercontent.com/17185640/173198772-7a85996d-86a3-4196-acae-1f81531ed933.png)

This general proof is more familiar to A-Level students but we could show it as a proof by cases rather than by contradiction - again, the power of direct and indirect proofs can be talked about later. We could also show how to restate our proof by cases as one by contradiction.


## How powerful is Euclid's proof?

![How powerful is Euclid's theorem?](https://user-images.githubusercontent.com/17185640/173198790-fcd80012-e493-42e0-8673-c52545ddddd3.png)

This is when we would first talk about the power of proofs and questions about primes further than just how many there are - this would include things like counting the primes up to $n$ (and the twin prime conjecture?). We would be talking about why Euclid's proof leads to these things later.


## Erdős' proof (simplified?)

![(Simplified) Erdős' theorem](https://user-images.githubusercontent.com/17185640/173198811-314ed962-a473-418c-bd8b-1820dc70703c.png)

This is where my understanding ends - I would be very excited to learn from you. From what I can understand, Erdős proves there are infinite primes by proving there are more than $\log _2 \sqrt{n}$ primes up to $n$. This involves the fundamental theorem of arithmetic but you might be able to simplify it?


## The power dynamic of Euclid's proof and Erdős' proof

![Power dynamic of Euclid's vs. Erdős'](https://user-images.githubusercontent.com/17185640/173198831-fedd3fd3-e42d-40b3-8076-eee4aa7c99ea.png)

This is when we would talk about the mathematical implications of the proofs and why they are relevant to modern work, including any we have referenced up until now. We would ask more questions and talk about why they are relevant too - this would hopefully lead to Dirichlet or simplified Euclid-Dirichlet proofs but again this is where I would need to learn more. From what I understand, we could ask how primes are distributed in general and in residue classes modulo $d$ and how answers to these would be powerful.


If any of this is appealing, please let me know because my ideas might not be pratical - I will have so much to learn about this if they are. I would love to script and produce a video with an expert such as yourself. I have experience scripting and producing because I talked about [the maths and physics of music](https://youtu.be/5BxS12cvCKI) in Summer of Math Exposition 1. I do need to develop my production skills because I ran out of time to produce all the visuals or experiment with Manim animation last year but I will have the time this year. I'd also love to compose some music for the video I do this year but that's just a personal interest.

**Comment by InigoMontoya314** [comment]
Hi, I'm from #124 and #136. From my perspective(someone who just graduated from high school but have relatively good math background, so I've heard of Dirichlet's theorem), I think one way to make it work for a student like me is to go for special cases like d=1, and even that proof is quite challenging for me. There could be easier proofs that I'm not aware of, so please let me know if that's the case.

**Comment by Rami5743** [comment]
@[ajayberriman](https://github.com/ajayberriman),
Thank you very much for your interest. It seems that we have rather different things in mind, so I do not think that it will be fretful to work on this together. However, if you have any mathematical question on the subject or on  a related topic, I'll be happy to try to answer it.

**Comment by Rami5743** [comment]
@[InigoMontoya314](https://github.com/InigoMontoya314),
It depends on what do you mean by Dirichlet theorem. The standard meaning is that there are infinitely many primes in the sequence. In this case, d=1 is just Euclide theorem, though it is very butiful I'd hardly call it challenging. 

Dirichlet actually proved a stronger statement: the diversions of $$\sum\frac{1}{p}$$ when p is in the progression. The case p=1 becomes now Euler result, and ofcores any farther discussion of Dirichlet theorem have to start with it. There  are more quantitative version of the Direchlet theorem, the strongest of which being the PNT for arithmetic progression, here the case d=1 is just the PNT, which is at list as chalnging as the Dirichlet theorem and deserve a treatment by its own.

**Comment by ajayberriman** [comment]
No problem Rami. I do look forward to seeing what you make even if it goes right over my head 😄.

**Comment by Rami5743** [comment]
@all, 
Let me explain what do I mean by simplified Erdos  argument:
Assume that there are only finitely many primes: $p_1,\dots p_k$, then since any number can be decompose into prime factors (note that we do not  use the uniqueness here) there are no more then $\prod \log_{p_i}n$ numbers up to $n$. This means that $n \leq (log_2 n)^k$  which implies $log_2 n \geq n^{\frac{1}{k}}$, this leads to a contradiction.

This in fact gives the following bound:  $$\pi(n) \geq \frac{\ln(n)}{\ln(log_2 n)}.$$

This is a slightly simplified version of an argument by Erdos that gives the  slightly stronger bound $$\pi(n) \geq \frac{\log_2(n)}{2}.$$

Theas bounds are much weaker then the reality (given by the PNT) however they are way better then the bounds that can be obtained from original Euclid argument. This argument attributed to Erdos, so if Euler was awer of it he did not publish it, however it  seems to me natural to consider this argument as transitional stage between Euclid and Euler. Namely, 
While in this argument we count all numbers below n, in Euler's product we use the same method to count all numbers but giving the number $k$ weight $k^{-s}$.
